lib/shopify-cli/commands/help.rb

```diff
@@ -21,27 +21,46 @@ def call(args, _name)
         end
 
         preamble = <<~MESSAGE
-          CLI to help build Shopify apps faster.
-
           Use {{command:#{ShopifyCli::TOOL_NAME} help <command>}} to display detailed information about a specific command.
 
-          {{bold:Available commands}}
+          {{bold:Available core commands:}}
+
         MESSAGE
         @ctx.puts(preamble)
 
-        visible_commands = ShopifyCli::Commands::Registry
-          .resolved_commands
-          .select { |_name, c| !c.hidden }
-          .sort
+        core_commands.each do |name, klass|
+          next if name == 'help'
+          @ctx.puts("{{command:#{name}}}: #{klass.help}\n")
+        end
+
+        return unless inside_supported_project?
 
-        visible_commands.each do |name, klass|
+        @ctx.puts("{{bold:Project: #{File.basename(Dir.pwd)} (#{project_name})}}")
+        @ctx.puts("{{bold:Available commands for #{project_name} projects:}}\n\n")
```

Comment:
Minor quibble - `project_name` might be interpreted to allude to the name of the project (as provided by the developer) whose directory you're actually in, vs. the name of the project type of the project whose directory you're in!  So, maybe `project_type_name`?

Reply:
Yup, i agree. Fixed in 899da5e

Reply:
👍